code_indexer_loop/test_api.py

```diff
@@ -2,7 +2,7 @@
 
 import pytest
 
-from code_indexer_loop.code_splitter import (CodeSplitter,
+from src.code_splitter import (CodeSplitter,
```

Comment:
Why not use package name?

Reply:
This was a mistake and changed back to package name in final